tests/golden_tests/desugar_file/combinators.hvm

```diff
@@ -0,0 +1,23 @@
+(foo) = λa λb λc (foo a)
+
```

Comment:
Can you add this example: `(clax) = (@x x @a @b @c @d (clax d))`
I expect it to be converted to this
```
(clax) = (@x x clax$C0)
(clax$C0) = @a @b @c @d (clax d)
```